src/screens/EventView.js

```diff
@@ -52,7 +53,7 @@ class EventView extends Component {
 
                   <li className="uleventdatali" >
                      <text className="lisTxt_lead">Total Attending:</text>
-                     <text className="lisTxt_content">1s / </text>
+                     <text className="lisTxt_content">{this.state.attendees} </text>
```

Comment:
We don't want to list out the people who are attending. If you want to use `this.state.attendees` to get the # of attending then use `this.state.attendees.length` although it would be better to not store the attendees in state. We should just store the count in the state

Reply:
this can't be changed to do this until we can start assigning people to events. I did add a note so that we can remember to change it when we do have that ability.

Reply:
if you initialize attendees to an empty array, we can just use the `this.state.attendees.length` and the count will always be zero until we can start assigning people to events

Reply:
good suggestion, changed